mysql/datadog_checks/mysql/statements.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import copy
+import logging
+from contextlib import closing
+
+import pymysql
+
+from datadog_checks.base.utils.db.sql import compute_sql_signature
+from datadog_checks.base.utils.db.statement_metrics import StatementMetrics, apply_row_limits
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+
+logger = logging.getLogger(__name__)
+
+
+METRICS = {
```

Comment:
```suggestion
STATEMENT_METRICS = {
```
Is it possible to rename it like so it's more explicit?